pkg/api/types.go

```diff
@@ -584,10 +584,28 @@ const (
 	ClaimLost PersistentVolumeClaimPhase = "Lost"
 )
 
+type HostPathType string
+
+const (
+	// If nothing exists at the given path, an empty directory will be created there. Otherwise, behaves like HostPathExists
+	HostPathUnset HostPathType = ""
+	// If nothing exists at the given path, the pod will fail to run and provide an informative error message
+	HostPathExists HostPathType = "exists"
+	// If a file does not exist at the given path, the pod will fail to run and provide an informative error message
```

Comment:
@thockin According to the proposal, both `BlockDev`  and `CharDev` are treated as device. There is no need to separate them. So we only have to handle `Device`. Any comments?